src/components/Header/Navbar.js

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+const HamburgerButton = styled.button`
+    border: none;
+    border-radius: 2px;
+    height: 30px;
+    width: 30px;
+    background-color: transparent;
+    position: absolute;
+    top: 25px;
+    right: 20px;
+    z-index: 999;
+    span {
+        height: 3px;
+        background-color: #FFFFFF;
+        position: relative;
+        width: 28px;
+        margin: auto;
+        display: block;
+        border-radius: 4px;
+    }
+    span::before, span::after {
+        content: "";
+        height: 3px;
+        width: 28px;
+        background-color: #FFFFFF;
+        position: absolute;
+        display: block;
+        border-radius: 4px;
+    }
+    span::after {
+        top: calc(100% - 14px);
+    }
+    span::before {
+        top: 10px;
+    }
+    .active {
+        position:absolute;
+        top: 50%;
+        right: 50%;
+        transform: translate(50%, -50%) rotate(45deg);
+    }
+    .active::before, .active::after {
+        position:absolute;
+        top: 50%;
+        right: 50%;
+        transform: translate(50%, -50%) rotate(270deg);
+    }
+
+    @media (min-width: 895px) {
+        display: none;
+}
+`
+const MenuBackground = styled.div`
+    display: ${({ open }) => { return open ? 'block' : 'none' }};
+    width: 66vw;
+    position: absolute;
+    top: 0px;
+    right: 0;
+    background-color: #000;
+    opacity: 85%;
+    height: 100vh;
```

Comment:
Is it intentional to have the navbar dropdown go all the way down to the buttom of the screen? If not, maybe you could alter the height to a smaller value?